tests/integration/targets/ec2_instance/aliases

```diff
@@ -0,0 +1,4 @@
+cloud/aws
+shippable/aws/group3
```

Comment:
Is it worth bringing this over given there's another PR to purge shippable references?

Reply:
Ah no, I haven't landed that one in community yet - good catch thanks!